management/change_function.py

```diff
@@ -4,7 +4,7 @@
 # Make sure the player doesn't already have the given role, for this function resets abilities like how many amulets the amulet holder has.
 def swap(rolename):
     # TODO
-    pass
+    return True
```

Comment:
Should this not be returning a warning if the function is deprecated?

Reply:
The function will most likely be deleted soon. If not, yes, we should.